packages/foundations/scss/helpers/_font.scss

```
@@ -1,9 +1,8 @@
 @mixin set-component-variables($size) {
 	/* Those variables are only for components to calculate heights and change icons */
-	--db-base-icon-font-size: var(--db-base-icon-font-size-#{$size});
-	--db-base-icon-font-family: var(--db-base-icon-font-family-#{$size});
+	--icon-font-weight: var(--db-base-icon-weight-#{$size});
+	--icon-font-size: var(--db-base-icon-font-size-#{$size});
```

```suggestion
	--db-icon-font-weight: var(--db-base-icon-weight-#{$size});
	--db-icon-font-size: var(--db-base-icon-font-size-#{$size});
```

Let's "scope" all of the CSS variables that we introduce by `db`.

---

fixed